compiler/testData/diagnostics/tests/extensions/additionalReceivers/resolveCallWithExplicitReceiver.kt

```diff
@@ -0,0 +1,17 @@
+open class A
```

Comment:
I think it'd be nice to have some more tests: 
-Additional receiver has a type `T` when `T` is a type parameter
-Additional receiver has a type `List<T>` when `T` is a type parameter